.circleci/config.yml

```diff
@@ -24,9 +24,6 @@ jobs:
       - run:
           name: Install dependencies
           command: make bootstrap
-      - run:
```

Comment:
This was removed because OpenShift support was taken out of aks-engine, but I think it should stay in acs-engine. (For now--I'm going to remove OpenShift support separately in a later PR.)

Reply:
@CecileRobertMichon thoughts? Because we don't run OpenShift tests anymore this is fine to take out preemptively.

Reply:
It's ok with me, I'm just complaining that the PR would be simpler without this removal, and that the OpenShift code removal contained herein is incomplete, so if we merge this I'll follow up with another PR.

Reply:
I'm fine with it either way, I would have preferred removing it with the PR that removes the relevant code but unit tests passed already so this is fine too.